libs/shared/ui/src/lib/utils/toast-error.tsx

```diff
@@ -1,6 +1,23 @@
+import { type IconName } from '@fortawesome/fontawesome-common-types'
 import { type SerializedError } from '@qovery/shared/utils'
 import toast, { ToastEnum } from './toast'
 
-export function toastError(error: SerializedError | Error, title?: string, message?: string): void {
-  toast(ToastEnum.ERROR, title || error.name || 'Error', message || error.message || 'No message found')
+export function toastError(
+  error: SerializedError | Error,
+  title?: string,
+  description?: string,
```

Comment:
Perhaps we need to keep our fallback for `title` and `description` ?

Reply:
It is still here, line 16 and 17

Reply:
My bad thanks!